frontend/workflows/projectCatalog/src/details/info/index.tsx

```diff
@@ -14,7 +14,7 @@ interface ProjectInfoProps {
 }
 
 const StyledRow = styled(Grid)({
-  marginBottom: "8px",
+  marginBottom: "16px",
```

Comment:
Why change this value?

Reply:
The margin change is to keep parity. There is a default margin change on the grid (-4 -> -8) so I modified this to fix the spacing. 

For the spacing change, it was necessary before, doesn't seem to be anymore...so I'm reverting it.